apps/sample-app/src/App.tsx

```diff
@@ -0,0 +1,87 @@
+import './App.css';
+
+import { Spinner, SpinnerSize } from '@fluentui/react';
+import { Button } from '@fluentui/react-components';
+import {
+  FluentProvider,
+  teamsDarkTheme,
+  teamsHighContrastTheme,
+  teamsLightTheme,
+  Theme,
+} from '@fluentui/react-components';
+import { app, authentication } from '@microsoft/teams-js';
+import React, { useState } from 'react';
+
+import { ProfileContent } from './components/Profile';
+const App: React.FC = () => {
+  const [isInitialized, setIsInitialized] = React.useState(false);
+  const [accessToken, setAccessToken] = React.useState<string>();
+
+  React.useEffect(() => {
+    (async () => {
+      await app.initialize();
+      setIsInitialized(true);
+    })();
+  }, [setIsInitialized]);
+
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  async function handle() {
+    const result = await authentication.authenticate({
+      url: 'https://localhost:4003/?auth=1',
+    });
+    setAccessToken(result);
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [context, setContext] = useState<app.Context>();
+  const [currTheme, setCurrTheme] = useState<Theme>(teamsLightTheme);
```

Comment:
the initial theme should be taken from `getContext` and later optionally updates via `registerOnThemeChangeHandler`